src/components/home/Service.js

```diff
@@ -1,7 +1,32 @@
+"use client";
 import React from "react";
+/*
+  Componenets in Service Card:
+  icon
+  name
+  description
+  socialMedia
+*/
 
-const Service = () => {
-  return <div>Service</div>;
+const Service = ({ icon, name, description, socialMedia }) => {
+  return (
+    <div className="bg-winc-yellow w-11/12 h-96 rounded-lg flex items-center justify-center">
```

Comment:
don't hard code height. remove h-96

Reply:
I can't change the height to 11/12 because it makes the rectangle weird looking